.github/workflows/create-crd-update-pr-in-longhorn-repo.yml

```
@@ -37,6 +37,14 @@ jobs:
         bash scripts/generate-longhorn-yaml.sh
         bash scripts/helm-docs.sh
 
+    - name: Get Head Commit Name
+      id: get_head_commit_name
+      run: echo "::set-output name=commit_name::$(git log -1 --pretty=format:'%an')"
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)